doc/manual/types.rst

```diff
@@ -120,8 +120,8 @@ using :func:`convert`:
 .. doctest:: foo-func
 
     julia> function foo()
-             x::Int8 = 100
-             x
+               x::Int8 = 100
+               x
            end
```

Comment:
Should all examples in the manual be indented with 4 spaces? I've noticed indentation can be rather inconsistent.